model/flow/ledger.go

```diff
@@ -87,8 +87,14 @@ func CadenceRegisterID(owner []byte, key []byte) RegisterID {
 }
 
 func NewRegisterID(owner, key string) RegisterID {
+	ownerString := ""
```

Comment:
better to add some comments

Reply:
added in [6f73ce9](https://github.com/onflow/flow-go/pull/4942/commits/6f73ce92ec4f45cad2ac7a616a1f85ecb18a1b82)